### ISSUE TITLE
How did you define novel?

### ISSUE THREAD

**Issue by dudududu12138** [alevar/chess3_construction#2]
Hi,
After assemble with stringtie, how did you define  novel transcripts that not in gencode or refseq database? Just by the matching of intron chain? Did you treat a transcript as novel if the class code of it is not "=" ?
Thanks! Looking forward to your reply.

**Comment by alevar** [comment]
Hi,

Yes, throughout the construction of the catalogue the intron chain compatibility ("=" code produced by gffcompare) was used to separate known from novel transcripts (with exception of single-exon transcripts which were processed separately and their novelty assignment might not be as reliable due to the differences at the 3' and 5' ends). In the final catalogue, this information is stored under the "db_xref" tag. If a transcript had assembly_id assigned and no db_xref tag, then it was treated as novel.

Hope this helps,

Ales

**Comment by dudududu12138** [comment]
> Hi,
> 
> Yes, throughout the construction of the catalogue the intron chain compatibility ("=" code produced by gffcompare) was used to separate known from novel transcripts (with exception of single-exon transcripts which were processed separately and their novelty assignment might not be as reliable due to the differences at the 3' and 5' ends). In the final catalogue, this information is stored under the "db_xref" tag. If a transcript had assembly_id assigned and no db_xref tag, then it was treated as novel.
> 
> Hope this helps,
> 
> Ales

Thanks! And I don't know if my understanding of your filtering methods is true, so I post below:

- Step1: You assembled the raw bam files with stringtie2 and got 26,335,900 transcripts.
- Step2: You aggregated the raw bam files among tissues and filtered coverage, then assembled with stringtie2.
- Step3: You extracted the intersection of step1 and step2, then you got 987,244 transcripts.

Is my undterstanding correct?

**Comment by alevar** [comment]
That is correct. The expression estimates, 3' and 5' ends, coverage, etc were also taken from the original ~27M assembly.
